src/tasks.ts

```diff
@@ -42,7 +44,9 @@ export interface Execution {
  */
 function createShellExecution(execution: Execution): ShellExecution {
   const { binary, command, args, cwd, env } = execution;
-  const cmdLine = `${command || binary} ${args.join(' ')}`;
+  const cmdLine = RLSConfiguration.loadFromWorkspace('').useWSL
```

Comment:
At a first glance it'd be better to modify the `Execution` before passing it to this function or introduce a `useWsl: boolean` argument here and propagate it downwards as needed